### ISSUE TITLE
How to use this

### ISSUE THREAD

**Issue by rpelletierrefocus** [rad10/OCR-to-CSV#22]
I would love to take this for a spin, but there is no information in the README regarding how to use this. Could you provide some instruction?

**Comment by rad10** [comment]
Yes. I have been working on a wiki page for the tool. It should hopefully be ready soon

**Comment by rad10** [comment]
I have added better documentation to the project. Give a comment if there are other issues.
